evennia/utils/eveditor.py

```diff
@@ -97,8 +97,10 @@ def _quitfunc(caller):
 
  :s <l> <w> <txt> - search/replace word or regex <w> in buffer or on line <l>
 
- :j <l> <w> - justify buffer or line <l>. <w> is f, c, l or r. Default f (full)
- :f <l>     - flood-fill entire buffer or line <l>. Equivalent to :j <l> l
+ :j <l> <a> =<w>  - justify buffer or line <l>. <a> is f, c, l or r. <w> is
+                    width. Default for <a> is l (left). Default for <w> is {_DEFAULT_WIDTH}
+ :f <l> =<w>      - flood-fill entire buffer or line <l> to width <w>.
+                    Equivalent to :j <l> l
```

Comment:
This is a backwards-incompatible change. I think we could just as well retain the old functionality as a default: 

```
:j <l> <a>  - working as before (fixed width, just change `<w>` to `<a>`)
:j <l> <a> = <w>  - using the = to show that we are introducing a width 
:f <l> - default flood fill
:f <l> = <w> - flood fill with width
```